deploy-board/deploy_board/templates/configs/metrics_config.tmpl

```diff
@@ -143,6 +163,11 @@
     var validUrl = false;
     var newEntry;
     $(function () {
+        $("select[name='metricsType']").change(function () {
+                document.getElementById("newMetricLabel").style.visibility= this.value=='Gauge' ? 'visible' : 'hidden';
```

Comment:
Use jquery selector rather than document getElementById for consistency. Or even better, use vuejs to binding the field